### DISCUSSION TITLE
Please update the docs with examples that use px

### DISCUSSION THREAD

**Discussion by hellowhirl** [tailwindlabs/tailwindcss#8476] [Ideas]
It isn't clear for properties like 'width" on how to specify px values. All we can see on the docs is:
```
w-0 | width: 0px;
w-px | width: 1px;
```
Please also refer to this Stack Overflow issue which answers this confusion:
https://stackoverflow.com/questions/65491795/how-can-i-specify-exactly-600px-width-in-tailwind-css

**Comment by jenstornell** [comment]
It's in the docs https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values

**Comment by adamwathan** [comment]
Hey! Our spacing scale uses `rem` by default because it lets your UI scale with the root font size if the user has set a custom font size in their browser preferences.

We show the pixel equivalents (based on a 16px default font size) of the `rem` values in the docs:

<img width="818" alt="image" src="https://user-images.githubusercontent.com/4323180/171284818-d71e7ca8-4ba0-4964-9669-b6cabe28d41e.png">

If you want to use pixel values instead, you have two options:

1. [Override your spacing scale](https://tailwindcss.com/docs/customizing-spacing#overriding-the-default-spacing-scale) with a custom one that uses pixels instead of `rem`.

2. Use [arbitrary values](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values) like `w-[11px]` to ignore the spacing scale and set the exact pixel value you want.
